MatrixSDK/VoIP/CallStack/MXCallStackCall.h

```diff
@@ -140,6 +140,28 @@ NS_ASSUME_NONNULL_BEGIN
              success:(void (^)(void))success
              failure:(void (^)(NSError *error))failure;
 
+#pragma mark - DTMF
+
+/**
+ Indicates whether this call can send DTMF tones.
+ This property will be false if the call is not connected yet.
+ */
+@property (nonatomic, readonly) BOOL canSendDTMF;
+
+/// Creates a task to send given DTMF tones in the call. If there is a task already running, it'll be canceled.
```

Comment:
Can we use the same comment style as the existing one is this file?

Reply:
👍